src/lib/utils.ts

```diff
@@ -417,3 +417,14 @@ export const getOrderItemQuantities = (items: PublicOrderItem[]): PublicOrderIte
 
   return Array.from(itemMap.values());
 };
+
+/**
+ * Validates src for event cover image, returning a default if invalid
+ * @param src src for cover image
+ * @returns a valid image src
+ */
+export const getDefaultEventCover = (src: any): string => {
+  if (!src || typeof src !== 'string' || !/^(http|\/)/.test(src))
+    return '/assets/graphics/store/hero-photo.jpg';
```

Comment:
Can we also do some suffix matching if that works also? For example, this is what we have for board photos on the main website
```tsx
  const boardPictureRegex = /(jpg)|(png)|(jpeg)/gim;
  const validProfileImage = profile_image.match(boardPictureRegex);
```

Reply:
donzo